### DISCUSSION TITLE
Can't connect litdb to local SQLite db in type safe way

### DISCUSSION THREAD

**Discussion by nikivdev** [litdb/litdb#4] [General]
Here is my repo: https://github.com/nikitavoloboev/litdb-test

I don't get how to connect litdb to SQLite in type safe way as trying out the code in docs breaks on import to the drive, unclear how to make use of it and how to do queries in type safe way.

**Comment by mythz** [comment]
Docs for using Bun SQLite is at: https://litdb.dev/bun-sqlite

E.g. you should resolve `connect` from the `@litdb/bun-sqlite` package:

```ts
import { connect } from "@litdb/bun-sqlite"
```
